betty/html/__init__.py

```diff
@@ -113,11 +114,30 @@ def cite(self, citation: Citation) -> int:
             return self._cited.index(citation) + 1
 
 
-class _Breadcrumb(Dumpable):
+@final
+class Breadcrumb(Dumpable):
```

Comment:
Make this implement `LinkedDataDumpable` instead, so we can generate relative URLs for HTML breadcrumbs and absolute ones for our JSON-LD.